secp256k1-sys/src/types.rs

```diff
@@ -43,13 +45,12 @@ impl AlignedType {
 #[cfg(all(feature = "std", not(rust_secp_no_symbol_renaming)))]
 pub(crate) const ALIGN_TO: usize = mem::align_of::<AlignedType>();
 
-
 #[cfg(test)]
 mod tests {
     extern crate libc;
-    use std::os::raw;
-    use std::mem;
     use std::any::TypeId;
+    use std::mem;
+    use std::os::raw;
```

Comment:
Seems this also should be replaced with `core`

Reply:
well these aren't in core :)

Reply:
Below in line 66 we have `core::mem`

Reply:
`any::TypeId` and `mem` are in `core` but `os::raw` isn't